build.gradle

```diff
@@ -4,6 +4,11 @@ buildscript {
 		mavenCentral()
 	}
 
+	ext.springBootVersion = "2.0.0.RELEASE"
+	ext.springCloudVersion = "Finchley.M9"
+	ext.lombokVersion = "1.16.8"
+	ext.springCloudNetflixVersion = "2.0.0.M7"
```

Comment:
Wondered if we should explicitly set the spring cloud netflix version like this, or import the Finchley BOM and have it dependency managed for us? Finchley.M9 specifies SCN 2.0.0.M8, so we're already behind here.

Reply:
Nice spot, removed that line as it wasn't used